securedrop_client/db.py

```diff
@@ -215,6 +215,18 @@ def seen_by(self, journalist_id: int) -> bool:
 
         return False
 
+    @property
+    def seen_by_list(self) -> Dict[str, 'User']:
```

Comment:
It looks like `make check` may be failing because these would be reformatted (by `black` I think?).